src/components/Tooltip/Tooltip.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div
+        ref="tooltip"
+        class="d-inline-block"
+    >
+        <slot v-if="!!$slots.default" />
+    </div>
```

Comment:
Only thing I don't like is the addition of an element. It will result in situations like:
```html
<tooltip class="w-100">
    <div class="w-100" />
<tooltip>
```
Could this not work?
```suggestion
    <component
        :is="is"
        ref="tooltip"
    />
```
Where `is` by default is a `div`?